subsystem/utils_test.go

```diff
@@ -480,6 +480,13 @@ func getDefaultInventory(cidr string) *models.Inventory {
 	return hwInfo
 }
 
+func getDefaultNutanixInventory(cidr string) *models.Inventory {
+	vmwareInventory := *getDefaultInventory(cidr)
+	vmwareInventory.SystemVendor = &models.SystemVendor{Manufacturer: "Nutanix", ProductName: "Nutanix", Virtual: true, SerialNumber: "3534"}
```

Comment:
```suggestion
	vmwareInventory.SystemVendor = &models.SystemVendor{Manufacturer: "Nutanix", ProductName: "AHV", Virtual: true, SerialNumber: "3534"}
```
to be closer to real-world values